build/jade.js

```diff
@@ -5,6 +5,9 @@ var find = require('find')
 var jade = require('jade')
 var async = require('async')
 var mkdirp = require('mkdirp')
+var extend = require('extend')
+var requireDir = require('require-dir')
+var helpers = requireDir('../helpers')
```

Comment:
What happens if this directory doesn't exist?

To be safe, can we use `path.join(__dirname, '..', 'helpers')` here instead?